### PR TITLE
Feature/requirejs

### PR DESCRIPTION
**Pull request by marcqualie**

### CODE REVIEW

**Comment by scottrobertson** [comment]
Javascript seems broken for me. None of the bootstrap things work at all (dropdowns, modals). @marcqualie is getting 404's on jquery, but i am not.

**Comment by marcqualie** [comment]
I think the 404 may have been due to some chrome extension or something as the javascript works on the project view page.

**Comment by outrunthewolf** [comment]
I'm looking into this now

**Comment by outrunthewolf** [comment]
I dropped out bootstrap js to lighten the amount of 3rd party javascript we were using, moved the dropdowns to CSS and the modals are using a simple toggle function now. 

If you see us using more of the bootstrap functionality in the future I can add it back in, but otherwise it seemed a bit pointless to keep the whole thing for the 2 small features we were using it for.

**Comment by scottrobertson** [comment]
Looks awesome. Good to merge from what i can see.

File: app/Criterion/UI/View/Blocks/AddProject.twig [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div id="addProject" class="modal hide fade" tabindex="-1" role="dialog" aria-labelledby="addProjectLabel" aria-hidden="true">
+<div id="addProject" class="modal" style="display: none" tabindex="-1" role="dialog" aria-labelledby="addProjectLabel" aria-hidden="true">
```
Comment (marcqualie) [inline app/Criterion/UI/View/Blocks/AddProject.twig]: Not sure about the inline styling :P

**Comment by marcqualie** [comment]
:+1: boom!

**Pull request merged**